docs/version-specific/generic_easyblocks.rst

```diff
@@ -233,7 +233,7 @@ Build a Python package and module with cmake.
     Some packages use cmake to first build and install C Python packages
     and then put the Python package in lib/pythonX.Y/site-packages.
 
-    We install this in a seperate location and generate a module file 
+    We install this in a separate location and generate a module file 
```

Comment:
@ocaisa This is a typo in a docstring of the generic `CMakePythonPackage` easyblock, should also be fixed there or it'll be re-introduced (all docs under `docs/version-specific` are re-generated automatically with every EasyBuild release)